src/utils/bitreader.h

```diff
@@ -2,13 +2,15 @@
 #define BITREADER
 #include <stdint.h>
 #include <stdlib.h>
```

Comment:
`stdlib.h` er deprecated i C++, brug `cstdlib` i stedet for.

Reply:
fixed